src/components/LineChart/LineChart-redux.tsx

```diff
@@ -1,37 +1,21 @@
 import React from 'react';
-import { useMappedState, useDispatch } from 'redux-react-hook';
+import { useMappedState } from 'redux-react-hook';
 
 import LineChartContainer from './LineChart-container';
-import { getBlockchainByDatasource } from '../../store/reducers/dataSource';
 import { Block, State} from "../../types";
 
-import { LOADER_STATE } from '../../store/actions/loader';
 
 const LineChartRedux = () => {
     const mapState = (state: State): { blokchain: Block[] } => ({
-        blokchain: getBlockchainByDatasource(state, state.dataSource)
+        blokchain: []
```

Comment:
Is it not supposed to be set to ethereum blockchain for now?

Reply:
Will do in Line Chart data fetch task